server/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -873,8 +880,24 @@ private void ensureOpen() {
     public final TransportStats getStats() {
         MeanMetric transmittedBytes = outboundHandler.getTransmittedBytes();
         MeanMetric readBytes = inboundHandler.getReadBytes();
+        MeanMetric inboundRequests = inboundHandler.getInboundRequests();
+        MeanMetric outboundResponses = outboundHandler.getOutboundResponses();
+        MeanMetric inboundKeepAlivePings = inboundHandler.getInboundKeepAlivePings();
+        MeanMetric outboundKeepAlivePongs = outboundHandler.getOutboundKeepAlivePongs();
+        TransportStats.InboundConnectionsStats inboundConnectionsStats = new TransportStats.InboundConnectionsStats(
```

Comment:
Maybe create a `Builder` for `TransportStats.InboundConnectionsStats`. This constructor already pretty much impossible to read without an IDE showing parameter names and it's just going to get worse as we add more to this object?